crud/insert.lua

```diff
@@ -98,4 +94,42 @@ function insert.call(space_name, obj, opts)
     }
 end
 
+--- Inserts an object to the specified space
+--
+-- @function object
+--
+-- @param string space_name
+--  A space name
+--
+-- @param table obj
+--  Object object (according to space format)
```

Comment:
`Object`?

Reply:
Fixed in https://github.com/tarantool/crud/compare/57d8d662f964250f6720c850a734721e730144ba..0583a496d5d16035d34ddad4447cd1ca862b8788